docs/components/connectors/out-of-the-box-connectors/rabbitmq-inbound.md

```diff
@@ -10,7 +10,7 @@ The **RabbitMQ Connector** is an inbound Connector that allows you to connect yo
 ## Prerequisites
 
 To use the **RabbitMQ Connector**, you need to have installed a RabbitMQ server and create the relevant [credentials](https://www.rabbitmq.com/passwords.html).
-It is highly recommended to use Camunda secrets to store credentials so you don't expose sensitive information directly from the process. See [this appendix entry](#how-do-i-store-secrets-for-my-connector) to learn more.
+It is highly recommended to use Camunda secrets to store credentials, so you don't expose sensitive information directly from the process. See [this appendix entry](#how-do-i-store-secrets-for-my-connector) to learn more.
```

Comment:
```suggestion
Using Camunda secrets to store credentials is highly recommended, so you do not expose sensitive information directly from the process. See [this appendix entry](#how-do-i-store-secrets-for-my-connector) to learn more.
```
A little rewording and removing the contraction (don't -> do not).

Reply:
Done